src/dictionary.c

```diff
@@ -330,6 +330,8 @@ SEXP vctrs_id(SEXP x) {
 // [[ register() ]]
 SEXP vctrs_match(SEXP needles, SEXP haystack, SEXP na_equal,
                  SEXP frame) {
+  struct r_lazy call = { .x = frame, .env = r_null };
```

Comment:
Some changes here in `dictionary.c` and `interval.c` are due to `vec_ptype2_params()` and `vec_cast_params()` getting a `call` argument